internal/go/skycfg/assert.go

```diff
@@ -137,18 +138,20 @@ func (t *TestContext) AssertBinaryImpl(op syntax.Token) func(thread *starlark.Th
 		passes, err := starlark.Compare(op, val1, val2)
 		if err != nil {
 			buf := new(strings.Builder)
-			thread.Caller().WriteBacktrace(buf)
+			stk := thread.CallFrame(1)
+			fmt.Fprintf(buf, "%s", stk)
```

Comment:
It looks like this `buf` isn't being used, so changing the error conditional to `if err != nil { return nil, err }` would clean it up.